staging/src/k8s.io/apiserver/pkg/server/httplog/httplog.go

```diff
@@ -218,6 +227,14 @@ func (rl *respLogger) Log() {
 		}
 	}
 
+	if pfc := request.PriorityAndFairnessClassificationFrom(rl.req.Context()); pfc != nil {
+		keysAndValues = append(keysAndValues,
+			"apf_pl", truncateLogField(pfc.PriorityLevelName),
+			"apf_fs", truncateLogField(pfc.FlowSchemaName),
+			"apf_d", pfc.FlowDistinguisher,
```

Comment:
I do not think these are intrinsically limited to 64 characters.  A username could be longer, right?